addons/mail_bot/static/src/js/notification_alert.js

```diff
@@ -8,15 +8,51 @@ var widgetRegistry = require('web.widget_registry');
 // Display Notification alert on user preferences form view
 // -----------------------------------------------------------------------------
 var NotificationAlert = Widget.extend({
-   template: 'mail.NotificationAlert',
-   /**
-    * @override
-    */
-   init: function () {
-      this._super.apply(this, arguments);
-      var hasRequest = this.call('mailbot_service', 'isRequestingForNativeNotifications');
-      this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
-   },
+    template: "mail.NotificationAlert",
+    /**
+     * @override
+     */
+    init() {
+        this._super(...arguments);
+        const hasRequest = this.call("mailbot_service", "isRequestingForNativeNotifications");
+        this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
+        if (this.isNotificationBlocked) {
+            if (navigator.userAgent.includes("Chrome")) {
+                this.settingUrl = "chrome://settings/content/siteDetails?site=https%3A%2F%2F" + window.location.host;
+            } else if (navigator.userAgent.includes("Firefox")) {
+                this.settingUrl = "about:preferences#privacy";
+            } else if (navigator.userAgent.includes("Safari")) {
+                this.settingPath = "(path:Safari > Preferences > Website > Notifications)";
+            }
+        }
+    },
+    /**
+     * Copies the browser settings URL link to the clipboard.
+     *
+     * @override
+     */
+    async start() {
+        await this._super(...arguments);
+        if (this.isNotificationBlocked && this.settingUrl) {
+            const $clipboardBtn = this.$(".o_clipboard_button").tooltip({
+                title: "Copied !",
+                trigger: "manual",
+                placement: "right"
+            });
+            const clipboard = new ClipboardJS($clipboardBtn[0], {
+                text: () => this.settingUrl,
+                container: this.el,
+            });
+            clipboard.on("success", function(e) {
+                clipboard.destroy();
+                setTimeout(() => {
+                    $clipboardBtn.tooltip("show");
+                    setTimeout(() => $clipboardBtn.tooltip("hide"), 800);
+                });
+            });
+            clipboard.on("error", e => clipboard.destroy());
```

Comment:
In which scenario this could emit an error? If it happens, I don't think we want to just destroy clipboard.